includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -766,7 +856,7 @@ protected function extract_response_body( $response ) {
 		// Check error codes for 4xx and 5xx responses.
 		if ( 400 <= $response_code ) {
 			if ( isset( $response_body['error'] ) ) {
-				$error_code    = $response_body['error']['code'];
+				$error_code    = isset( $response_body['error']['code'] ) ? $response_body['error']['code'] : '';
 				$error_message = $response_body['error']['message'];
 
 				if ( is_null( $error_code ) && ! empty( $response_body['error']['type'] ) ) {
```

Comment:
Good catch! It's best to default it to `null`, since that check was added just to avoid some notices when `code` was not present in the response. Fixed in https://github.com/Automattic/woocommerce-payments/pull/758/commits/73b8b9f650f4b41ecd7b4427b9b4811753d9a206.